src/styles/oeb/oeb_styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '@angular/cdk/overlay-prebuilt.css';
+// @import '@angular/cdk/overlay-prebuilt.css';
```

Comment:
I think this would be needed for the ```dialog``` or ```alertdialog``` components from spartan, should we leave it in? Or add it when we add those components?